config/v1/types_infrastructure.go

```diff
@@ -1741,6 +1741,7 @@ type NutanixPlatformSpec struct {
 	// prism element clusters to improve fault tolerance of the cluster.
 	// +listType=map
 	// +listMapKey=name
+	// +openshift:validation:FeatureGateAwareMaxItems:featureGate=NutanixMultiSubnets,maxItems=32
```

Comment:
Did you intend for this to be 32? Previously we discussed 256, I'm happy with 32 if you are though

Reply:
Yes, I think 32 should be big enough.